api/office_structure/schema.py

```diff
@@ -9,6 +9,7 @@
     validate_empty_fields,
     validate_parent_node_id,
     )
+from utilities.validator import validate_tag_id
```

Comment:
Great work @TeamoreA, Please move this function to the `utilities.validations` since that's where the validations are located.

Reply:
I was facing a problem of cyclic imports by importing this `from api.tag.models import Tag as TagModel` in the file `utilities.validations.py` that why I moved it.

Reply:
Ok, I suggest you use `verify_tag_id` instead of `validate_tag_id` since you are checking to see whether the id exists or not.